application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableModerationAction.java

```diff
@@ -12,21 +12,14 @@
  */
 interface RevocableModerationAction {
     /**
-     * Classification of a revocation failure.
+     * The name of the action, that will be logged in case of a failed revocation.
+     *
+     * <p>
+     * Naming convention examples: {@code "ban"}, {@code "mute"}, {@code "quarantine"}
```

Comment:
The example would be better if it gives an example of a full sentence, to give the context in which the word might be used.